### ISSUE TITLE
Predicted score vectors do NOT seem to be satisfying

### ISSUE THREAD

**Issue by YoungjunLee13** [gcorso/DiffDock#170]
**I'm evaluating the loss function and also directly comparing predicted / true score vectors for the best model ('workdir/paper_score_model/best_ema_inference_epoch_model.pt').** This comparison is to gauge how closely the score model approximates the true score function. The loss values I obtained were around 0.5 and 0.6 for the training and validation sets, respectively.

Initially, these values seemed sufficiently low, but a direct comparison of the predicted score vectors with the true score vectors raises some doubts. **There were sometimes considerable differences in their magnitudes, and often their directions were not aligned.** For instance, with the aforementioned model (batch size of 2), the results were as follows. I am curious whether this level of similarity is adequate for the score model to be effective in an inference scheme. **Could you possibly share the loss values and predicted score vectors that you obtained for this model, and, if available, the entire learning graph?**
```
tr_pred
tensor([[ 7.7381, -4.6792,  7.1045],
        [ 0.0882, -0.0388,  0.2425]], device='cuda:0', grad_fn=<DivBackward0>)
tr_score
tensor([[ 3.6262,  0.1902,  3.5643],
        [ 0.0626, -0.0292,  0.0917]])
rot_pred
tensor([[  7.9226, -23.1543,  10.0307],
        [ -0.6370,   0.6247,  -0.9663]], device='cuda:0',
       grad_fn=<MulBackward0>)
rot_score
tensor([[-11.5837,  -1.3307, -11.5590],
        [ -0.8421,   0.4747,  -1.0578]])
tr_loss
tensor(0.2656, grad_fn=<MeanBackward1>)
rot_loss
tensor(0.5786, grad_fn=<MeanBackward1>)
```
```
tr_pred
tensor([[ 0.4043, -1.2220, -1.3719],
        [ 0.0767, -0.0267, -0.0593]], device='cuda:0', grad_fn=<DivBackward0>)
tr_score
tensor([[ 0.6455, -1.4594, -2.1916],
        [ 0.0539, -0.0398, -0.0506]])
rot_pred
tensor([[-4.7778, -0.4370, -4.4685],
        [-0.0554, -0.0620,  0.0457]], device='cuda:0', grad_fn=<MulBackward0>)
rot_score
tensor([[-2.6724,  0.7735, -2.4689],
        [-0.0971,  0.0567,  0.1158]])
tr_loss
tensor(0.1122, grad_fn=<MeanBackward1>)
rot_loss
tensor(0.8070, grad_fn=<MeanBackward1>)
```
